web/src/compose_state.ts

```diff
@@ -7,6 +7,7 @@ import * as sub_store from "./sub_store";
 
 let message_type: "stream" | "private" | false = false;
 let recipient_edited_manually = false;
+let last_focused_compose_type_input: HTMLInputElement | false = false; // an HTML input element or false-y
```

Comment:
Why did you pick `false` rather than `null` or `undefined` as the unset state?

Reply:
Yep I based this off that convention; have added starting a discussion on this to my todo list